app/services/doc_auth/lexis_nexis/responses/lexis_nexis_response.rb

```diff
@@ -0,0 +1,104 @@
+module DocAuth
+  module LexisNexis
+    module Responses
+      class LexisNexisResponse < DocAuth::Response
+        PII_DETAILS = [
+          'Age'.freeze,
```

Comment:
maybe easier to call `.map(&:freeze).freeze`?

Reply:
Nice.